spec/controllers/api/v1/bands_controller_spec.rb

```diff
@@ -17,4 +17,17 @@
         end
     end
 
+    describe "GET#show" do
+        it "should return one band" do
+
+            get :show, params: {id: band.id}
+            returned_json = JSON.parse(response.body)
+
+            expect(response.status).to eq 200
+            expect(response.content_type).to eq("application/json; charset=utf-8")
+
+            returned_json["name"] = band.name
```

Comment:
this last piece needs to be in an expect statement! (and would need to be double equals anyway at that!)